specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -567,17 +591,27 @@
    "outputs": [],
    "source": [
     "hdd_annual = compute_annual_aggreggate(\n",
-    "    data=hdd, \n",
+    "    data=hdd_aggregated, \n",
     "    name=\"Annual Heating Degree Days (HDD)\", \n",
     "    num_grid_cells=num_grid_cells\n",
     ")\n",
     "cdd_annual = compute_annual_aggreggate(\n",
-    "    data=cdd, \n",
+    "    data=cdd_aggregated, \n",
```

Comment:
I think something is going wrong in how cdd_annual interacts w/ compute_multimodel_stats - the plot in 4e shows a multimodel min that's lower than any of the individual model outputs

Reply:
@gdicecco I have an answer! This is in fact the minimum, because FGOALS is uniquely across the board the minimum. If you toggle off the minimum line in the figure, FGOALS is directly underneath it. This also explains why the trend line is shifted seemingly downwards when looking at the other 3 models too. I also confirmed this by looking at cdd_aggregated too, and FGOALS is consistently the lowest value so this does track

Worth putting 1-2 lines of markdown to clarify this, or is that opening a can of worms about disregarding some models? 
![Screen Shot 2023-03-23 at 10 02 47 AM](https://user-images.githubusercontent.com/106171203/227228096-742abf68-f05f-472c-93fb-0739c06df2b1.png)
![Screen Shot 2023-03-23 at 10 02 56 AM](https://user-images.githubusercontent.com/106171203/227228098-b21c50c7-09fc-4e80-9d65-dd10a82644b8.png)
![Screen Shot 2023-03-23 at 10 09 14 AM](https://user-images.githubusercontent.com/106171203/227229847-4ea144bb-f513-431c-8376-02483ad0fff8.png)

Reply:
@thenaomig this is the issue I was referring to in standup - do we want to just alter the visualization a bit so its clear the fgoals and multimodel min lines are completely overlapping? Address in markdown? This plot I think will raise questions about fgoals in general so I'm unsure how we want to proceed.

Reply:
Ideally we'd do this with bias-corrected data. (At the same time, it is concerning that it's so very biased with respect to the other simulations. We should discuss leaving it out of the catalog in future.) For now, I'd add something to the markdown about the fact that the gridded data is not currently bias-corrected and that as a result the min or max could just reflect a simulation that is biased high or low compared to the others.